services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -137,4 +142,27 @@ class FlowHelper {
         return new SimpleDateFormat("ddMMMHHmmss_SSS", Locale.US).format(new Date()) + "_" +
                 faker.food().ingredient().toLowerCase().replaceAll(/\W/, "")
     }
+
+    /**
+     * Checks flow rules presence (or absence) on source and destination switches.
+     */
+    private void checkRulesOnSwitches(FlowPair<Flow, Flow> flowEntry, int timeout, boolean cookiesPresent) {
+        def cookies = [flowEntry.left.cookie, flowEntry.right.cookie]
+        def switches = [flowEntry.left.sourceSwitch, flowEntry.left.destinationSwitch].toSet()
+        switches.each { sw ->
+            Wrappers.wait(timeout) {
+                try {
+                    def result = northbound.getSwitchRules(sw).flowEntries*.cookie.containsAll(cookies)
+                    assert cookiesPresent ? result : !result
```

Comment:
can be simplified to 
assert !(cookiesPresent ^ result)

(just a neat little trick)